### ISSUE TITLE
Double-Click sorting is still active, even when disabled in the config

### ISSUE THREAD

**Issue by Landaz2** [kyrptonaught/Inventory-Sorter#59]
Pretty much the title. I disabled all ways of sorting so that only clicking the ui button should do the sorting. However, double clicking still sorts the inventory that is clicked in.

Minecraft 1.18
Fabric 0.12.8
Inventory Sorting 1.8.0-1.18

config.json:

```
{
	// Enable 'Sort' button in inventorys
	"displaySort": true,
	// Middle click slot to sort inventorys
	"middleClick": false,
	// Double click slot to sort inventorys
	"doubleClickSort": false,
	// Enable second 'Sort' button in player inv
	"seperateBtn": true,
	// Sorting inv also sorts player inv
	"sortPlayer": false,
	// Method of sorting, NAME,CATEGORY,MOD
	"sortType": "NAME",
	// Display Sort Button Tooltip
	"displayTooltip": true,
	// Sort Inventory key
	"keybinding": "key.keyboard.unknown",
	// Should sort half of open inv highlighted by mouse
	"sortMouseHighlighted": false,
	"debugMode": false
}
```

**Comment by kyrptonaught** [comment]
I have found the issue and will release a fix shortly

**Comment by kyrptonaught** [comment]
This was released with 1.8.1
